packages/python/build.sh

```diff
@@ -98,7 +99,7 @@ termux_step_create_debscripts() {
 
 	if [[ -f "$TERMUX_PREFIX/bin/pip" && \
 	 ! (("$TERMUX_PACKAGE_FORMAT" = "debian" && -f $TERMUX_PREFIX/var/lib/dpkg/info/python-pip.list) || \
-	    ("$TERMUX_PACKAGE_FORMAT" = "pacman" && -d $TERMUX_PREFIX/var/lib/pacman/local/python-pip-*)) ]]; then
+	    ("$TERMUX_PACKAGE_FORMAT" = "pacman" && $(ls $TERMUX_PREFIX/var/lib/pacman/local/python-pip-*))) ]]; then
```

Comment:
Dollar sign (`$`) for command substitution (`$(...)`) is not escaped. Rather I suppose subshell (`(...)`) should be used instead.

Reply:
I mean, the current `postinst` is not what is expected:

```bash
#!/data/data/com.termux/files/usr/bin/bash

if [[ -f "/data/data/com.termux/files/usr/bin/pip" && 	 ! (("debian" = "debian" && -f /data/data/com.termux/files/usr/var/lib/dpkg/info/python-pip.list) || 	    ("debian" = "pacman" && )) ]]; then
echo "Removing pip..."
rm -f /data/data/com.termux/files/usr/bin/pip /data/data/com.termux/files/usr/bin/pip3* /data/data/com.termux/files/usr/bin/easy_install /data/data/com.termux/files/usr/bin/easy_install-3*
rm -Rf /data/data/com.termux/files/usr/lib/python3.11/site-packages/pip
rm -Rf /data/data/com.termux/files/usr/lib/python3.11/site-packages/pip-*.dist-info
fi

echo ""
echo "== Note: pip is now separate from python =="
echo "To install, enter the following command:"
echo "   pkg install python-pip"
echo ""

exit 0
```

Reply:
Ok, thanks for saying

Reply:
Ah this is in `[[...]]`. Neither would work.

Reply:
This conditional should be refactored. Almost unreadable for me.